app/main.py

```diff
@@ -1,3 +1,44 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from typing import Any
+from pathlib import Path
+from app.customer import Customer, Car
+from app.shop import Shop
+
+
+def load_config() -> dict:
+    path_json = Path(__file__).parent.joinpath("config.json")
+    with open(path_json, "r") as file_info:
+        return json.load(file_info)
+
+
+def shop_trip() -> Any:
+    list_cust = load_config()["customers"]
+    customers = []
+    for cost in list_cust["customers"]:
+        customer = Customer(
+            cost["name"], cost["product_cart"], cost["location"], cost["money"],
+            Car(cost["car"]["brand"], cost["car"]["fuel_consumption"])
+        )
+        return customers.append(customer)
```

Comment:
It is a little strange your code always end in this line

Reply:
remove return keyword
```suggestion
       customers.append(customer)
```